### PR TITLE
User id in JWT

### PR DESCRIPTION
**Pull request by ekivolowitz**

### CODE REVIEW

File: star_tides/api/util/issue_jwt.py [hunk 1]
```diff
@@ -5,16 +5,15 @@
...
     now = datetime.now(timezone.utc)
-
     jwt_token = jwt.encode(
         {
             'iss': 'star-tides',
             'exp': now + timedelta(hours=1),
             'iat': now,
             'claims': {
-                'email': email
+                'id': str(user_id)
```
Comment (LOZORD) [inline star_tides/api/util/issue_jwt.py:16]: Why not have both the email and the ID in the claims? Does it complicate (or simplify) any code to have both?

File: star_tides/api/util/issue_jwt.py [hunk 2]
```diff
@@ -5,16 +5,15 @@
 from flask import current_app
 
 
-def create_jwt(email):
+def create_jwt(user_id):
```
Comment (LOZORD) [inline star_tides/api/util/issue_jwt.py:8]: Can you annotate the type on this to make usages easier to read please?

**Review by LOZORD — APPROVED** [review]
Ready to merge once you address my comments.

File: star_tides/api/util/issue_jwt.py (hunk 1 above)
Comment (ekivolowitz) [reply star_tides/api/util/issue_jwt.py:16]: Because it'll be passed on every single network request, I want to keep this lean

**Pull request merged**